### DISCUSSION TITLE
how to integrate an existing tee-based PPC system into secretflow?

### DISCUSSION THREAD

**Discussion by qsmen** [secretflow/secretflow#285] [General]
in secretflow,  spu supports many kinds of backend, like MPC, HE and TEE.  As for MPC and HE, as they are software implementation, we can use them directly.  But for TEE, as it is a hardware implementation, I want to know the requirement on TEE when integrating an existing TEE-based PPC system into secretflow?

**Comment by Chrisdehe** [comment]
This discissions has been transferred to #287
